website/docs/r/mysql_server.html.markdown

```diff
@@ -50,34 +57,16 @@ The following arguments are supported:
 
 * `sku` - (Required) A `sku` block as defined below.
 
+* `storage_profile` - (Required) A `storage_profile` block as defined below.
+
+* `create_mode` - (Optional) The mode to create a new server, supported values `Default` or `PointInTimeRestore`. 
```

Comment:
> The mode to create a new server, supported values `Default` or `PointInTimeRestore`.

For consistency purposes can we make this "The Creation Mode of the MySQL Server, used when restoring a database backup. Possible values include `Default` and `PointInTimeRestore`"